CHANGELOG.rst

```diff
@@ -12,13 +12,25 @@ This project adheres to `Semantic Versioning`_ starting with version 0.11.0.
 
 .. note:: This version is not yet released and is under active development.
 
-
-[1.7.0] - 2020-01-16
```

Comment:
why? dont' you want to cut the release today?

Reply:
Originally I wanted to leave the topmost entry of the changelog at 1.6.X, because we might not always know when we are going to release the next minor release. But given that it should be today I guess it makes sense to bump it now to 1.7.